Microsoft.Azure.Cosmos/src/ChangeFeedHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+
+namespace Microsoft.Azure.Cosmos
+{
+    using Microsoft.Azure.Documents;
+
+    internal static class ChangeFeedHelper
+    {
+        internal static bool isChangeFeedWithQueryRequest(
```

Comment:
Method is not private, so the casing can be upper camel case

```suggestion
        internal static bool IsChangeFeedWithQueryRequest(
```